.mailmap

```diff
@@ -214,3 +214,4 @@ ZhouSky <11711923@mail.sustech.edu.cn>
 Vincent Gagnon <vincentgagnon53@gmail.com>
 Tom Warnke <Toromtomtom@users.noreply.github.com>
 Eric Lau <919023+skeric@users.noreply.github.com>
+Raphael Kreft <raphaelkreft@gmx>
```

Comment:
All right, mine is "r.kreft@unibas.ch" and my full name: Raphael Kreft
I really don't know why it used my second account here, since i never used that account to work on this project ...